static/app/utils/getCookie.tsx

```diff
@@ -1,17 +1,13 @@
-/**
- * Get the value of a cookie by name
- */
 export default function getCookie(name: string): string | null {
-  if (!document.cookie || document.cookie === '') {
-    return null;
+  if (document.cookie && document.cookie !== '') {
+    const cookies = document.cookie.split(';');
+    for (let i = 0; i < cookies.length; i++) {
+      const cookie = cookies[i].trim();
+      // Does this cookie string begin with the name we want?
+      if (cookie.substring(0, name.length + 1) === name + '=') {
+        return decodeURIComponent(cookie.substring(name.length + 1));
+      }
+    }
   }
-
-  const cookies = document.cookie.split(';');
-  const cookie = cookies.find(c => c.substring(0, name.length + 1) === `${name}=`);
```

Comment:
FYI it is because we weren't trimming here, the cookie has spaces at the FRONT of it.

Reply:
didn't we add a `trim()` here? https://github.com/getsentry/sentry/pull/39185/files#r978335242